### DISCUSSION TITLE
[Discussions] What is quality parameter in case of `OutputFormat.png`

### DISCUSSION THREAD

**Discussion by maRci002** [fluttercandies/flutter_image_editor#112] [General]
### Content

As far as I know JPEG uses a lossy compression algorithm with a quality parameter, while PNG uses a lossless compression algorithm with a compression level parameter.

0 means no compression applied and 9 means best compression but the quality remains the same, so my question how does quality paramter works?

**Comment by CaiJingLong** [comment]
This plug-in uses native api and does not use a third-party library.
Currently the quality of the png format of the android/iOS sdk is ignored.

In the android: https://android.googlesource.com/platform/frameworks/base/+/956e4f969f5188b45dfea1ef776137a85658159f/graphics/java/android/graphics/Bitmap.java#1374

In iOS:

https://developer.apple.com/documentation/uikit/1624096-uiimagepngrepresentation?language=objc

In macOS:

https://developer.apple.com/documentation/appkit/nsimage/1519949-tiffrepresentationusingcompressi#parameters

The NSImage convert to png also will ignore compression factor.

**Reply by CaiJingLong** [reply]
It's not that I don't want to, but the API doesn't support.

**Reply by maRci002** [reply]
That's fine in that case add some documentation which mentions that the qualtiy parameter is ignored in case of PNG.
